colossalai/zero/utils/zero_hook.py

```diff
@@ -92,18 +68,32 @@ def pre_bwd_exec(self, module: torch.nn.Module, input, output):
             for param in module.parameters(recurse=False):
```

Comment:
You can remove this branch now, since `self._stateful_tensor_mgr` can't be `None`.

Reply:
Sure.